src/js/components/CheckBox.js

```diff
@@ -21,30 +42,37 @@ export default class CheckBox extends Component {
         );
       }
     }
+
     if (this.props.className) {
       classes.push(this.props.className);
     }
 
-    return (
-      <label className={classes.join(' ')}
-        aria-describedby={this.props.ariaDescribedby}
-        aria-lebelledby={labelId}>
+    let children = [(
+      <span>
```

Comment:
can you add `key="checkbox"` to this span to avoid react warnings for unique components inside an array?